_pages/welcome.md

```diff
@@ -0,0 +1,27 @@
+---
+permalink: /welcome/
```

Comment:
You can delete the permalink and rename the file to index.md and put it into the root directory where index.html is located. Afterwards delete the index.html file. And finally, can you test locally that it is working?